src/Umbraco.Web.UI.Client/.eslintrc

```
@@ -19,10 +19,10 @@
         "tinyMCE": false,
         "FileReader": false,
         "Umbraco": false,
+        "umb": false,
```

@nathanwoulfe seems you missed the new name here:

```suggestion
        "Utilities": false,
```